modules/scanning-images.adoc

```diff
@@ -70,68 +69,85 @@ Scanner can check for vulnerabilities in images that use the following package f
 * dpkg
 * RPM
 
-[discrete]
+[id="supported-programming-languages_{context}"]
 == Supported programming languages
 
 Scanner can check for vulnerabilities in dependencies for the following programming languages:
 
+* Go (supported in Scanner V4 only)
 * Java
 * JavaScript
+* NodeJS (supported in Scanner V4 only)
 * Python
 * Ruby
```

Comment:
* Go (Scanner V4, only)
  * Binaries. The standard library version used to build the binary is analyzed. If built with module support (go.mod), then the dependencies are are analyzed as well
* Java
  * JAR/WAR/EAR
* JavaScript/Node.js
  * npm pacakge.json
* Python
  * egg and wheel formats
* Ruby
  * gem